apps/guardian-ui/src/components/RunDKG.tsx

```diff
@@ -56,7 +56,7 @@ export const RunDKG: React.FC<Props> = ({ next }) => {
             setError(`${t('run-dkg.error-config')}`);
             break;
           default:
-            setError(`${t('run-dkg.error-default')} '${status}'`);
+            setError(`${t('run-dkg.error-default')} '${JSON.stringify(status)}'`);
```

Comment:
Thanks for the first time contribution @harsh9539! We don't actually want the whole object to be rendered here, it's quite large and contains many irrelevant fields. We just want the `status.server`.

```suggestion
            setError(`${t('run-dkg.error-default')} '${status.server}'`);
```

Reply:
I changed the file according to your need.